graphitron-codegen-parent/graphitron-java-codegen/src/test/java/no/sikt/graphitron/queries/fetch/OutputTest.java

```diff
@@ -214,11 +261,31 @@ void outerNestedWithArgument() {
     void outerNestedWithArgumentListed() {
         assertGeneratedContentContains(
                 "outerNestedWithArgumentListed", Set.of(CUSTOMER_TABLE),
-                "ctx, List<String> firstName,",
-                "CustomerTable::new))).from(_a_customer)" +
-                        ".where(firstName.size() > 0 ? _a_customer.FIRST_NAME.in(firstName) : DSL.noCondition())" +
-                        ".orderBy(_a_customer.fields(_a_customer.getPrimaryKey().getFieldsArray()))",
-                "new Outer(_iv_it)))).fetchOne"
+                """
+                            public static Outer queryForQuery(DSLContext _iv_ctx, List<String> firstName, SelectionSet _iv_select) {
+                                var _a_customer = CUSTOMER.as("customer_2168032777");
+                                return _iv_ctx
+                                        .select(queryForQuery_outer(firstName))
+                                        .fetchOne(_iv_it -> _iv_it.into(Outer.class));
+                            }
+                            private static SelectField<Outer> queryForQuery_outer(List<String> firstName) {
+                                var _a_customer = CUSTOMER.as("customer_2168032777");
+                                return DSL.row(
+                                        DSL.row(
+                                                DSL.multiset(
+                                                        DSL.select(queryForQuery_outer_d1_customers())
+                                                        .from(_a_customer)
+                                                        .where(firstName.size() > 0 ? _a_customer.FIRST_NAME.in(firstName) : DSL.noCondition())
+                                                        .orderBy(_a_customer.fields(_a_customer.getPrimaryKey().getFieldsArray()))
+                                                )
+                                        ).mapping(_iv_e -> _iv_e.map(Record1::value1))
+                                ).mapping(Functions.nullOnAllNull((_iv_it) -> new Outer(_iv_it)));
+                            }
+                            private static SelectField<CustomerTable> queryForQuery_outer_d1_customers() {
+                                var _a_customer = CUSTOMER.as("customer_2168032777");
+                                return DSL.row(_a_customer.getId()).mapping(Functions.nullOnAllNull(CustomerTable::new));
+                            }
+                        """
```

Comment:
```suggestion
                "queryForQuery(DSLContext _iv_ctx, List<String> firstName,",
                ".select(queryForQuery_outer(firstName)).fetchOne",
                "queryForQuery_outer(List<String> firstName)",
                "DSL.multiset(DSL.select(queryForQuery_outer_d1_customers()).from(_a_customer)" +
                ".where(firstName.size() > 0 ? _a_customer.FIRST_NAME.in(firstName) : DSL.noCondition())" +
                ".orderBy(_a_customer.fields(_a_customer.getPrimaryKey().getFieldsArray()))"
```

Nøyaktig samme som over, men nå er argumentet listed, så vi må ta med et par listegreier i tillegg.